src/utils/file_manager.h

```diff
@@ -0,0 +1,47 @@
+//
+// Created by blinku on 15.12.18.
+//
+
+#pragma once
+
+#include "pch.h"
+
+namespace Bald::Utils {
+
+/**
+* @class FileManager
+* @brief implementation of file manager system
+*/
+
+    class FileManager {
+    public:
+        enum Size : char {
+            SMALL, BIG
+        };
+    public:
+
+        /**
+         * @fn FileManager
+         * @brief default constructor
+         */
+        FileManager() = default;
```

Comment:
We shouldn't be ablle to create object of this type

Reply:
done